Bieker2020-libs

ArchaeOphelie opened the pull request:
Added PRJNA339123, data previously published in Sanchez Barreiro 2017. I found some discrepancies between the data on ENA and the supplementary methods, so I've kept the data from ENA. I've put unknown for what I couldn't positively identify from the paper.

# Pull Request

This PR is for a

- [X] [New Publication(s)](#new-publication)
- [ ] [Correction](#correction)

For the following list(s):

- [ ] ancientmetagenome-anthropogenic ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-anthropogenic))
- [ ] ancientmetagenome-environmental ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-environmental), ask **@thedir-team-dirt** for advice)
- [X] ancientmetagenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientmetagenome-hostassociated), ask **@thedir-team-bugparty** for advice)
- [ ] ancientsinglegenome-hostassociated ([README](https://github.com/SPAAM-workshop/AncientMetagenomeDir/tree/master/ancientsinglegenome-hostassociated), ask **@thedir-team-pathopeeps** for advice)

## New Publication

### Publication Information

This pull request is to add samples from the following publication(s): 10.1111/1755-0998.13174

This is to close <!-- REPLACE this comment (including the < > symbols) with a hashtag and the corresponding issue number here, e.g. #10 -->

### Checklist

- [X] Publication is published (preprints currently not accepted)?
- [X] Checked the publication is not already in the database?
- [ ] Checked samples in this publication are not previously published _data_ (newly re-sequenced metagenomes are OK!)?
- [X] Samples are shotgun metagenomes (hostassociated-singlegenome may also contain whole-genome enriched data)?
- [X] Checked the list follows conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)?
- [ ] Changelog is updated to include the publication under 'Added'?
- [ ] Pull request has passed validation checks (see automated comment from `github-bot`)?
- [X] Review requested from corresponding team (see list of lists above for each team)?
- [ ] (If applicable) Made a separate PR to add new categories to controlled lists (stored under `/assets/enums`, e.g. material or archive)? If so the corresponding PR is here: <!-- REPLACE this comment  (including the < > symbols) with that PR -->

<details>
  <summary>If you do not know how to check errors in failed validation checks, expand here</summary>
</details>

## Correction

This PR is for

- [ ] ancientmetagenome-hostassociated
- [ ] ancientsinglegenome-hostassociated
- [ ] ancientmetagenome-environmental
- [ ] ancientmetagenome-anthropogenic

### Reference

This pull request is to correct samples from the following publication(s): <!-- REPLACE this comment (including < >  symbols) with the publication being corrected -->

This is to close <!-- REPLACE this comment (including the < > symbols) with a hashtag and the corresponding issue number here, e.g. #10 -->


### Description

The issue is:

<!-- REPLACE this comment (including < >  symbols) and justification of the correction -->

### Checklist

- [ ] Checked the corrected entries follow conventions as described in the corresponding sample type's README file (e.g. using ERS/SRS accession codes for ENA/SRA)
- [ ] Changelog is updated to include the publication under 'Changed'?
- [ ] Pull request has passed validation checks (see automated comment from `github-bot`)?
- [ ] Review requested from corresponding team (see list of lists above for each team)?

jfy133 commented:
I think there is a typo with the project_name?

ArchaeOphelie commented:
Heya @jfy133,

Following amending the other paper this morning, I decided to look back through this one too just to double check what I did yesterday. All the samples (38 samples, 52 libraries) added for this project ID (ENA:PRJNA339123) are from Sanchez Barreiro 2017, as in 21 samples (21 libraries) had previously been sequenced by Sanchez Barreiro and therefore only repooled and sequenced by Bieker2020, and 17 samples (31 libraries) were redone for this study (as in the NGS libraries were rebuilt). 

I went back to check the samples table, and currently all 17 samples that have been redone by Bieker2020 are there. However, the other 21 samples are not. I also couldn't find them under Sanchez Barreiro (in case those samples had been entered through that original paper). 
Therefore we should probably add them?

Finally, thank you for spotting the typo! I had added it as a self-reference and forgot to remove it before uploading the data. I'll remove it now and resolve the conflict/rerun the checks.

ArchaeOphelie commented:
Hello again @jfy133 

There appears to be an issue with the DOI here. The checks come back as duplicate error with the following two DOIs, though only one corresponds to Bieker2020:

10.1111/1755-0998.13174 - that is Bieker2020 herbaeum paper
10.1371/journal.pgen.1007155  - that is a paper by Exposito-Alonso. 

Not sure how best to proceed?

jfy133 reviewed (changes requested):
Hi @ArchaeOphelie 

Sorry it's taken such a long time to get to this.

Please don't include additional samples/libraries that are not listed in the sample list, as these are not listed as being used in Bieker2020 (they are rather from the original host-genome analysis publication of the same sample).

Please make sure to only include libraries listed in RAW, and double check that you don't have any extras over what is reported in Table 1 of the Bieker 2020 publication, as presumably any extra libraries NOT listed in this table, were not metagenomically anlaysed.

Otherwise, please check your archive_sample_accession column, these should be the SRS codes `secondary_sample_accession` column of the RAW table.

Accordingly, please cross-reference the ERR codes of TAble 1 of the SI, as I think you only want the libraries suffixed with `_WGS`.

Also there seems to be somethign wrong on the ENA, it says it is SINGLE end sequenced but provides two FASTQ files :thinking: I think it will be better to record per the number of FASTQ files... `librar_layout` column MAY NOT correspond to what you put in the main table here...

If this is unclear plesae message me on slack (this is indeed a bit messy)

ArchaeOphelie commented:
Hello @jfy133 
Notes for this pull request:
1)  We have an extra sample in ST1 of the publication named USNH-39. That one also has the library prep method from Sánchez Barreiro 2020, but there appears to have been 2 sequencing rounds.  That sample is in our samples table but associated with another archive project PRJEB34825 and its corresponding ERS code ERS3882395. That corresponds to the first sequencing round. The second sequencing round is associated with this archive project (PRJNA339123) and an SRR code SRR4047228. So the sample is present in our table, just not linked to this archive project. I have added the library metadata for that sample, as agreed
2) We have removed all the libraries of these samples which were labelled 'samplename _10' and 'samplename_100' and kept those recorded as 'samplename_WGS', as we are aiming for entire genome reconstruction.
Thank you again for all your help sorting this! Good luck with the final stretch!

jfy133 reviewed (approved):
Thanks very much! wierd metadata is single end but available files are paried end...
